docs/Administrator Guide/Geo Replication.md

```diff
@@ -38,192 +25,131 @@ geo-replication:
+For example,
 
-        # gluster system:: execute mountbroker opt mountbroker-root /var/mountbroker-root
-        # gluster system:: execute mountbroker opt geo-replication-log-group geogroup
-        # gluster system:: execute mountbroker opt rpc-auth-allow-insecure on
+```
+# gluster-mountbroker setup /var/mountbroker-root geogroup
+```
 
-5.  In any one of the Slave node, Add Mountbroker user to glusterd vol file using,
+In any one of Slave node, Run the following commands to add Volume and
+user to mountbroker service.
 
-        # gluster system:: execute mountbroker user geoaccount slavevol
+```
+gluster-mountbroker add <VOLUME> <USER>
+```
 
-    where slavevol is the Slave Volume name
+For example,
 
-    If you host multiple slave volumes on Slave, for each of them and add the following options to the volfile using,
+```
+# gluster-mountbroker add gvol-slave geoaccount
+```
 
-        # gluster system:: execute mountbroker user geoaccount2 slavevol2
-        # gluster system:: execute mountbroker user geoaccount3 slavevol3
+(**Note**: To remove a user, use `gluster-mountbroker remove` command)
 
-    To add multiple volumes per mountbroker user,
+Check the status of setup using,
 
-        # gluster system:: execute mountbroker user geoaccount1 slavevol11,slavevol12,slavevol13
-        # gluster system:: execute mountbroker user geoaccount2 slavevol21,slavevol22
-        # gluster system:: execute mountbroker user geoaccount3 slavevol31
+```
+# gluster-mountbroker status
+```
 
-6.  Restart `glusterd` service on all Slave nodes.
+Restart `glusterd` service on all Slave nodes.
 
 ## Setting Up the Environment for Geo-replication
 
-**Time Synchronization**
+### Time Synchronization
+
+On bricks of a geo-replication master volume, all the servers' time
+must be uniform. You are recommended to set up NTP (Network Time
+Protocol) service to keep the bricks sync in time and avoid the
+out-of-time sync effect.
 
--   On bricks of a geo-replication master volume, all the servers' time
-    must be uniform. You are recommended to set up NTP (Network Time
-    Protocol) service to keep the bricks sync in time and avoid
-    out-of-time sync effect.
+For example: In a Replicated volume where brick1 of the master is at
+12.20 hrs, and brick 2 of the Master is at 12.10 hrs with 10 minutes
+time lag, all the changes in brick2 between this period may go
+unnoticed during synchronization of files with Slave.
 
-    For example: In a Replicated volume where brick1 of the master is at
-    12.20 hrs and brick 2 of the master is at 12.10 hrs with 10 minutes
-    time lag, all the changes in brick2 between this period may go
-    unnoticed during synchronization of files with Slave.
+### Password-less SSH
 
-**Password-less SSH**
 Password-less login has to be set up between the host machine (where
-geo-replication Create command will be issued) and one of the Slave node
+geo-replication Create command will be issued) and one of the Slave
+node.
```

Comment:
...... and one of the Slave node for the unprivileged account created above.

Reply:
Done